docker/README.md

```diff
@@ -5,15 +5,17 @@ for testing & development purposes.
 
 ### Building 
 
-First, you need to build `teleport:latest` Docker image. This image is built
-when you type `make build` BUT...
+First, you need to build `teleport:latest` Docker image.
 
-But you have to build the base image first, by running `make docker`
-from `$GOPATH/github.com/gravitational/teleport` (repository base dir).
+Run the following commands from `$GOPATH/github.com/gravitational/teleport` (repository base dir):
 
-### Starting 
+```bash
```

Comment:
Can you also update the main Readme.md with these changes https://github.com/gravitational/teleport#building-teleport

Reply:
Added a docker section to root README: https://github.com/gravitational/teleport/pull/3400/commits/c22eec54c5762a118687c3a633c3473f472b967b